.bamboo/E2E-tests/execute-postgres.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+# Create libs folder because the Artemis docker compose file expects the .war file there
+mkdir -p build/libs
+mv ./*.war build/libs/
+
+# pass current host's hostname to the docker container for server.url (see docker compose config file)
+export HOST_HOSTNAME=$(hostname)
+
+cd docker
+#just pull everything else than artemis-app as we build it later either way
+docker compose -f cypress-E2E-tests-postgres.yml pull artemis-cypress postgresql nginx
+docker compose -f cypress-E2E-tests-postgres.yml build --build-arg WAR_FILE_STAGE=external_builder --no-cache --pull artemis-app
+docker compose -f cypress-E2E-tests-postgres.yml up --exit-code-from artemis-cypress
```

Comment:
Maybe it's better to add an argument for the docker compose .yml files and then add either ``cypress-E2E-tests-mysql.yml`` or ``cypress-E2E-tests-postgres.yml`` as argument in bamboo? But not 100 percent sure.